<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>305</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>K1USN SST logger</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>561</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(94, 92, 100)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>11</y>
       <width>551</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>JetBrains Mono</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(66, 66, 66);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>7</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Band:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="band_selector">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>7</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <item>
     <property name="text">
      <string>160</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>7</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>UTC:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="mycallEntry">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>7</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>Enter YOUR callsign.</string>
    </property>
    <property name="maxLength">
     <number>14</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Your Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genLogButton">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>202</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 53, 102);</string>
    </property>
    <property name="text">
     <string>Generate Logs</string>
    </property>
   </widget>
   <widget class="QLabel" name="radio_icon">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>7</y>
      <width>36</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="QRZ_icon">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>7</y>
      <width>29</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 26, 26);</string>
    </property>
    <property name="text">
     <string>QRZ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exchange_entry">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>202</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Exchange</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="callsign_entry">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>202</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>Enter other operators callsign.</string>
    </property>
    <property name="statusTip">
     <string>Enter other operators callsign.</string>
    </property>
    <property name="whatsThis">
     <string>Enter other operators callsign.</string>
    </property>
    <property name="maxLength">
     <number>14</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>CallSign</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>40</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>QSO's:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Total_CW">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>60</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mults:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Mults">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>80</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Score">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>80</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="myexchangeEntry">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>7</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>Your State</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Your Exchange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_gear">
    <property name="geometry">
     <rect>
      <x>694</x>
      <y>202</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/gear16x16.png</normaloff>icon/gear16x16.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="dupe_indicator">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>202</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(230, 97, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="utctime">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>7</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>00/00 00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>237</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F2">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>237</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F3">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>237</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F6">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>237</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F5">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>237</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F4">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>237</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F11">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>272</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>272</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F12">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>272</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F8">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>272</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F10">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>272</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F9">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>272</y>
      <width>103</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>F9</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
